scripts/blogView.js

```diff
@@ -49,32 +49,45 @@ blogview.handleCategoryFilter = function() {
 
```

Comment:
Can you see a way to refactor this function, similarly to the Author version? Could there then be a possible way to combine them?